packages/core/android/src/oldarch/src/DdLogs.kt

```diff
@@ -0,0 +1,163 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.reactnative
+
+import com.facebook.react.bridge.Promise
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.bridge.ReactContextBaseJavaModule
+import com.facebook.react.bridge.ReactMethod
+import com.facebook.react.bridge.ReadableMap
+
+/**
+ * The entry point to use Datadog's Logs feature.
+ */
+class DdLogs(
+    reactContext: ReactApplicationContext
+) : ReactContextBaseJavaModule(reactContext) {
+
+    private val implementation = DdLogsImplementation()
```

Comment:
Question: here I'm not using `lazy` to initialize the implementation, because I think the `lazy` init of the `Logger` inside `DdLogsImplementation` is enough, is it correct?

Reply:
depends on the need why `lazy` was put in the first place. Was it placed to avoid having NoOp implementation?

Reply:
Yes exactly

Reply:
should be enough then I believe